frontend/public/components/factory/table-filters.ts

```diff
@@ -265,6 +265,7 @@ export const tableFilters: TableFilterMap = {
     const status = volumeSnapshotStatus(snapshot);
     return statuses.selected.has(status) || !_.includes(statuses.all, status);
   },
+  'node-disk-name': (name, disks) => fuzzyCaseInsensitive(name, disks?.path),
```

Comment:
This filter is needed on lso and ocs pages as well, hence added in all filters.
Also found no other place to contribute.  May be passing a filter function in table or list component would help as well.
cc @rawagner